src/main/java/frc/robot/subsystems/scoring/AimerIOTalon.java

```diff
@@ -126,12 +129,81 @@ public void setFF(double kS, double kV, double kA, double kG) {
         aimerRight.getConfigurator().apply(slot0);
     }
 
+    private boolean checkForAimMotorFailure() {
+
+        /*if (!aimerLeft.isAlive() || !aimerRight.isAlive()) {
+            if (!motorCheckOverriden) setFF(ScoringConstants.aimerFaultkS, ScoringConstants.aimerFaultkV, ScoringConstants.aimerFaultkA, ScoringConstants.aimerFaultkG);
+            motorFailure = true;
+        } else if (motorFailure) {
+            if (!motorCheckOverriden) setFF(ScoringConstants.aimerkS, ScoringConstants.aimerkV, ScoringConstants.aimerkA, ScoringConstants.aimerkG);
+            motorFailure = false;
+        }
+        return motorFailure;*/
+
+        // LEFT MOTOR
+        if ((!aimerLeft.isAlive())
+                || // if motor disconnected
+                (Math.abs(aimerRight.getStatorCurrent().getValueAsDouble())
+                                        - Math.abs(aimerLeft.getStatorCurrent().getValueAsDouble())
+                                > 1
+                        && Math.abs(aimerLeft.getMotorVoltage().getValueAsDouble()) > 0.5)
+                || // if motor voltage is really small when it shouldn't be
+                (Math.abs(aimerLeft.getStatorCurrent().getValueAsDouble())
+                                        - Math.abs(aimerRight.getStatorCurrent().getValueAsDouble())
+                                > 1
+                        && Math.abs(aimerLeft.getMotorVoltage().getValueAsDouble())
+                                < 0.5) // if motor voltage is really large when it shouldn't be
+        ) {
+            motorLeftFailure = true;
+        }
```

Comment:
Since we are duplicating a lot of this code below, could we make this into a function that returns a boolean on if the motor is faulty or not?